addon/components/active-link.js

```diff
@@ -2,23 +2,28 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   tagName: 'li',
-  childLinkViews: [],
   classNameBindings: ['active'],
 
   active: Ember.computed('childLinkViews.@each.active', function() {
-    return Ember.A(this.get('childLinkViews')).isAny('active');
+    return this.get('childLinkViews').isAny('active');
   }),
 
-  didRender: function() {
+  init() {
+    this._super( ...arguments );
```

Comment:
There is no need to call super here. `init` on Ember.Component is an empty function.